spec/dummy/app/views/site/index.html.erb

```diff
@@ -4,10 +4,14 @@
 
 <div><%= t('site.index.footer') %></div>
 
+<div><%= link_to phrase('Phrase example page url'), example_path %></div>
```

Comment:
I'd move it to the bottom, just before the uneditable link so you have them logically near each other.